analyzer/runtime/extract.go

```diff
@@ -654,6 +656,7 @@ func extractEvents(blockData *BlockData, relatedAccountAddresses map[apiTypes.Ad
 					amount := &big.Int{}
 					amount.SetBytes(amountU256)
 					eventData.EvmLogName = apiTypes.Erc20Approval
+					eventData.EvmEventSignature = ethCommon.BytesToHash(event.Topics[0]).Hex()
```

Comment:
I would prefer to store the signature in a more normalized representation, i.e. raw bytes (`BYTEA` db type).

If we store a string
- we have to be more careful to support variants of search strings, e.g. lowercase vs uppercase hex, or present/absent leading `0x`
- we use up more space in the db
- it's easier to unknowingly store the wrong thing (like we were storing base64 until this PR -- thank you for finding that!)

If we want to the API to return the hex (more customary in eth) as opposed to base64 (recommended/prescribed by openapi for []byte fields), we can do the serialization in SQL in the SELECT to avoid type gymnastics in Go.

Reply:
Agreed, will update.

Reply:
Updated!